sdks/python/apache_beam/io/gcp/gcsio.py

```diff
@@ -150,6 +151,11 @@ def parse_gcs_path(gcs_path, object_optional=False):
   return match.group(1), match.group(2)
 
 
+def get_response_encoding():
+  """Determine whether to encode response content based on Python version."""
```

Comment:
Docstring suggestion (here and below): ```Encoding to use to decode HTTP response from Google APIs.```
Alternatively, you could inline this instead:
```
   ...
   http=get_new_http(),
   response_encoding=None if sys.version_info[0] < 3 else 'utf8')
```

Reply:
move it inline for simplicity. Thanks.